tests/utils/tools.php

```diff
@@ -391,10 +391,6 @@ function fromJSON($var) {
     return $func($var);
 }
 
-
-
-/* NOTE: Using this function will take down mongod ! */
```

Comment:
Oh, it's possible that getMore itself aborts (I don't recall offhand). I thought this was referring to the PHP function itself and sending the `configureFailPoint` command, which certainly wasn't true.